deployer/templates/fluentd.yaml

```diff
@@ -332,3 +336,11 @@ parameters:
   description: 'Configure MUX SERVER (false|true).'
   name: MUX_ALLOW_EXTERNAL
   value: "false"
+-
+  description: 'Fluentd buffer queue limit (0.12 only)'
```

Comment:
Add these defaults to openshift-ansible and please add appropriate notes with the unit (e.g. bytes, kb)